src/java.desktop/share/classes/sun/awt/image/ImageRepresentation.java

```diff
@@ -114,7 +114,8 @@ public synchronized void reconstruct(int flags) {
             try {
                 startProduction();
                 missinginfo = flags & ~availinfo;
-                while ((availinfo & ImageObserver.ERROR) == 0 &&
+                while ((availinfo &
+                        (ImageObserver.ERROR | ImageObserver.FRAMEBITS)) == 0 &&
                        missinginfo != 0)
```

Comment:
In my opinion, such formatting looks confusing, keeping both ERROR and FRAMEBITS on the same line improves readability even though it doesn't fit 80-column limit.
```suggestion
                while ((availinfo & (ImageObserver.ERROR | ImageObserver.FRAMEBITS)) == 0
                       && missinginfo != 0) 
```

Or alternatively, wrap `FRAMEBITS`:
```suggestion
                while ((availinfo & (ImageObserver.ERROR
                                     | ImageObserver.FRAMEBITS)) == 0
                       && missinginfo != 0)
```

Reply:
I agree the single line is one is more readable, was just following  the 80 column limit you told me before :-)